@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -315,21 +306,25 @@
   list-style: none;
   margin: 0;
   padding: 0;
-  width: 100%;
   display: inherit;
   flex-direction: inherit;
   gap: inherit;
   background-color: inherit;
   align-items: flex-start;
+  overflow: hidden scroll;
+  padding-inline: 0 var(--ax-spacing-4);
+
+  /* TODO: expected this to work
+*
+* scrollbar-gutter: stable;
```

Comment:
Is there a reason we need scrollbar gutter now?

Reply:
nothing is deemed needed atm, just a WIP to get scrollbars to work right atm (https://github.com/navikt/aksel/pull/3328#issuecomment-2457253720). Hope to get this done with _purely_ CSS changes as well. 💭

Reply:
Currently I reserve some space, but it looks ugly when they're not needed (asymmetrical). Might have to look ActionMenu later for some inspiration 👀 (assuming there is a similar issue there)